corehq/apps/registration/static/registration/js/register_new_user.js

```diff
@@ -71,12 +71,49 @@ $(function () {
         }
     });
 
-    // Handle phone number input
-    var $number = $('#id_phone_number');
-    $number.intlTelInput({
-        separateDialCode: true,
-        utilsScript: initial_page_data('number_utils_script'),
+// here, the index maps to the error code returned from getValidationError - see readme
+var errorMap = [ "Invalid number", "Invalid country code", "Too short", "Too long", "Invalid number"];
+
+// Handle phone number input
+var $number = $('#id_phone_number');
+var iti = $number.intlTelInput({
+    separateDialCode: true,
+    utilsScript: initial_page_data('number_utils_script'),
+    initialCountry: "auto",
+    geoIpLookup: function (success, failure) {
+        $.get("https://ipinfo.io", function () {}, "jsonp").always(function (resp) {
+            var countryCode = (resp && resp.country) ? resp.country : "";
+            success(countryCode);
+        });
+    },
+});
+    
+var input = document.querySelector("#id_phone_number"),
+  errorMsg = document.querySelector("#id_phone_number"),
+  validMsg = document.querySelector("#id_phone_number");
+
```

Comment:
setting everything as `#id_phone_number` is a temporary measure until I can get the error message to properly show up.